src/Admin/TimelineThemeAdmin.php

```diff
@@ -82,4 +157,28 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    /**
+     * @param Theme $object
+     */
+    public function postPersist($object)
```

Comment:
This class should use an `AlgoliaIndexedEntityAdminTrait` that handle an interface though a common manager to avoid adding these methods again and again.